src/pages/home/sidebar/SidebarScreen/BaseSidebarScreen.tsx

```diff
@@ -43,6 +56,27 @@ function BaseSidebarScreen() {
         updateLastAccessedWorkspace(undefined);
     }, [activeWorkspace, activeWorkspaceID]);
 
+    /**
+     * Navigate to scan receipt screen after it enabling camera permission from setting
+     * This will only works for ios application because we are saving last screen only for ios
+     */
+    useEffect(() => {
+        if (!lastScreen) {
+            return;
+        }
+        interceptAnonymousUser(() => {
+            updateLastScreen('');
+            IOU.startMoneyRequest(
```

Comment:
Please check this comment
https://github.com/Expensify/App/issues/46422#issuecomment-2277056712

Reply:
@ravindra-encoresky Please check this comment